x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/search/refine_potential_matches.ts

```diff
@@ -103,7 +102,7 @@ export const mostRecentCheckGroups = async (
       query: {
         bool: {
           filter: [
-            makeDateRangeFilter(queryContext.dateRangeStart, queryContext.dateRangeEnd),
```

Comment:
@andrewvc I think a defect is introduced here. I am running the patch with old heartbeat docs and I traced an exception to the line that runs these params against my es snapshot.

Reply:
Fixed, I was missing an `await`